### PR TITLE
Add documentation for Twitter API

### PR DESCRIPTION
**Pull request by mspensieri**

### CODE REVIEW

**Comment by spasiu** [comment]
enumerate Twitter here? https://github.com/smooch/smooch-docs/pull/302/files#diff-37c18dd958b6b54c1a030371cf03a368R4

**Comment by spasiu** [comment]
Also List Integrations method needs Viber and Twitter in the parameters table.

**Comment by spasiu** [comment]
And finally, do we need to talk about carousel and list message support for Twitter?

https://docs.smooch.io/rest/#post-message

https://docs.smooch.io/rest/#list-message

**Comment by spasiu** [comment]
meanwhile I'm going to take a couple unnecessary enumerations of channels out of some other spots in the docs...

**Comment by mspensieri** [comment]
> Also, List Integrations method needs Viber and Twitter in the parameters table.

It also needs mailgun and wechat... Man these docs...

**Comment by mspensieri** [comment]
> And finally, do we need to talk about carousel and list message support for Twitter?

I think it can fit under the "all other channels" definition. They don't have native support, but it's slightly better than a text fallback

File: slate/source/includes/rest/managed/_integrations.md [hunk 1]
```diff
@@ -63,6 +63,73 @@ In order to integrate a Facebook Messenger app you must acquire a Page Access To
...
+
+In order to create a Twitter DM integration you must acquire an Access Token Key and Secret from your user.
+These can be obtained via the [Twitter OAuth Flow](https://dev.twitter.com/oauth). Once you have acquired an access
+token pair from your user, call the Create Integration endpoint with your consumer key and secret, and the user’s
+access token pair.
+
+| **Arguments**             |   |
+|---------------------------|---|
+| **type**<br/><span class='req'>required</span> | The integration type: _twitter_. |
+| **consumerKey**<br/><span class='req'>required</span> | The consumer key for your Twitter app |
```
Comment (jpjoyal) [inline slate/source/includes/rest/managed/_integrations.md]: Any place we can link people for those keys ?

Comment (mspensieri) [inline slate/source/includes/rest/managed/_integrations.md]: There's a link to apps.twitter.com in the setup steps above (step 1)

**Pull request merged**